Source/WebInspectorUI/UserInterface/Images/Start.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
```

Comment:
Should we just rename `AuditStart.svg` to `Start.svg` so that we use a consistent "start" arrow for running things?

Reply:
There is actually a minor difference between the two.  `AuditStart.svg` (and all the other `Audit*.svg`) are rounded, whereas `Start.svg` (and `Stop.svg`, `Pause.svg`, etc.) are all pointed.  This was an intentional design choice back when the Audit Tab was first created to better give a sense of familiarity with `xctest`.  Perhaps we could revisit that decision in the future.